controllers/platform/services/services.go

```diff
@@ -408,6 +408,7 @@ func (j JobServiceHandler) ConfigurePersistence(containerSpec *corev1.Container)
 		c.Env = append(c.Env, persistence.ConfigurePostgreSQLEnv(p.PostgreSQL, j.GetServiceName(), j.platform.Namespace)...)
 		// Specific to Job Service
 		c.Env = append(c.Env, corev1.EnvVar{Name: "QUARKUS_FLYWAY_MIGRATE_AT_START", Value: "true"})
+		c.Env = append(c.Env, corev1.EnvVar{Name: "KOGITO_JOBS_SERVICE_LOADJOBERRORSTRATEGY", Value: "FAIL_SERVICE"})
```

Comment:
What's this new env? How does that relate to persistence? Is it something that we missed in previous PRs? 🤔

Reply:
This goes hand in hand with the jobs-service load procedure improvements for the GA.
Basically a new configuration that didn't exists before, and we add now. And yeah, to minimize the number of PRs it's included here.